extensions/scala3/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,13 @@
+name: "Scala 3"
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+metadata:
+  keywords:
+  - "scala"
+  - "scala3"
+  categories:
+  - "alt-languages"
+  status: "preview"
+  codestart:
+    name: "scala3"
+    kind: "core"
+    artifact: "io.quarkus:quarkus-project-core-extension-codestarts"
```

Comment:
Don't we need a specific codestart for the Scala projects? I would have expected it and I don't see anything in this PR?

Reply:
Ah yes, good catch. I think this was part of the original "find + replace all" I did. There was no codestart submitted with this, so this file probably should not have been committed.